opwen_email_server/celery/celery.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import
+from celery import Celery
+
+celery = Celery(include = [ "opwen_email_server.api.store_written_client_emails",
+                            "opwen_email_server.api.send_outbound_emails",
+                            "opwen_email_server.api.store_inbound_emails"])
+
+class Config:
+    BROKER_URL = "amqp://"
+    CELERY_RESULT_BACKEND = "amqp://"
```

Comment:
Given that none of our tasks are required to return anything, could you disable the results backend? This can be done via the [ignore_result](http://docs.celeryproject.org/en/latest/userguide/tasks.html#Task.ignore_result) configuration. This will simplify the system (since Celery now doesn't have to track return values) and also improve performance.

Reply:
Will do